torch/_inductor/codecache.py

```diff
@@ -268,18 +272,67 @@ def code_hash(code, extra=""):
     )
 
 
+def serialize_compiled_fx_arg(arg: Any):
+    if isinstance(arg, torch.fx.GraphModule):
+        # Special case for fx_graph
+        return str(arg).encode("utf-8")
```

Comment:
This is a bad way to hash the graph, we should have a TODO to make it better ;)

Note that @jansel in compiled backwards had to do something similar, although I don't think any of the code will be reusable here.